Kitodo-MODSXML-SchemaConverter/pom.xml

```diff
@@ -60,16 +50,22 @@
                 <artifactId>maven-jar-plugin</artifactId>
                 <version>${maven-jar-plugin.version}</version>
                 <configuration>
+                    <archive>
+                        <manifest>
+                            <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+                            <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+                        </manifest>
+                        <manifestEntries>
+                            <Implementation-Build-Date>${maven.build.timestamp}</Implementation-Build-Date>
+                        </manifestEntries>
+                    </archive>
```

Comment:
why do you add the timstamp to the manifest? Is it used by the serviceloader?

Reply:
Removed.